server/src/course/course.router.ts

```diff
@@ -69,3 +69,24 @@ courseRouter.post('/get-reviews', async (req, res) => {
       .json({ error: `Internal Server Error: ${err.message}` });
   }
 });
+
+/** Reachable at POST /api/courses/getPreDesc
+ * @body description: a course description
+ * Gets the processed description to use for the similarity algorithm
+ * Currently used for testing
+*/
+courseRouter.post('/getPreDesc', async (req, res) => {
```

Comment:
Would there be any errors to catch here? Also I think that a route name more like `/preprocess` or `/preprocess-desc` might fit more with our naming theme

Reply:
Yep we should handle errors at the router level